test/module/irohad/ordering/ordering_gate_test.cpp

```diff
@@ -161,26 +159,50 @@ TEST_F(OrderingGateTest, ProposalReceivedByGateWhenSent) {
   ASSERT_TRUE(wrapper.validate());
 }
 
+class QueueBehaviorTest : public ::testing::Test {
+ public:
+  QueueBehaviorTest(): ordering_gate(transport, 1, false) {};
+
+  void SetUp() override {
+    std::shared_ptr<OrderingGateTransport> transport =
```

Comment:
It will not be possible to set any expects on this `transport` pointer because gmock type is erased, so OrderingGateTransport -> MockOrderingGateTransport

Reply:
Also fixture member is not used, so it also will not be possible to set any expects in tests. Please reuse fixture member.